lib/lambda_ethereum_consensus/validator/validator.ex

```diff
@@ -146,6 +134,7 @@ defmodule LambdaEthereumConsensus.Validator do
   end
 
   defp handle_tick({slot, :second_third}, state) do
+    Logger.info("[Validator] #{state.validator.index} starting second third", slot: slot)
```

Comment:
Yes indeed, it was just here before the ready for review version. 

I already tackled both these and previous Logger.info that cluttered the logs before adding these ones. Almost every log in the Validator is now debug except for block publish and attest publish, which should happen just 1 time per epoch per validator. Also tried to homogenize Logging. This will probably change in the subsequent PRs though, but it allows for better debuging right now with low amounts of Validators.

Reply:
> A validator's state has a head_slot and head_root, which are shared by all. We might want to have them in the validator manager instead, and add those to the function calls. Not sure if that simplifies or complicates the code, but it deduplicates the variables.

Yes, totally agree, I'll be more explicit in the PR description about this, I'll gathered it upon the "it doesn't deal with some well-needed refactors", but yes, both head and slot as well as duty management should be shared. I didn't know either if it simplifies things or not but also agreed that's not the place for it. I just tried to touch ligic as fewer as possible in this PR, and add more logging.

> Maybe we should have the uninitialized validators in a separate variable instead of having them mixed and notifying them all. This is completely up to discussion and opinionated, but I do find it a bit weird to have a try_setup on every notification instead of trying that in the manager each time there's a change of target state.

Also agree, they are hard to track, up until now it wasn't an issue except for the validators just being there doing nothing, and as far as i could see the condition for them not being able to initialize is the lack of the public key in the validators set. I just ignore the calls in those cases for now instead of retrying the setup which wouldn't work in the new schema, holding all the validators until this ones correctly set up. I also think this is part of the deeper refactor.